testsuite/tests/singlecluster/authorino/collisions/auth_policy/test_collisions.py

```diff
@@ -0,0 +1,57 @@
+"""Test auth policies collisions when attached to the same target"""
+
+import pytest
+
+from testsuite.httpx.auth import HeaderApiKeyAuth
+from testsuite.kuadrant.policy import has_condition
+from testsuite.kuadrant.policy.authorization.auth_policy import AuthPolicy
+
+
+@pytest.fixture(scope="module")
+def api_key(create_api_key, module_label):
+    """Creates API key Secret."""
+    return create_api_key("api-key", module_label, "api_key_value")
+
+
+# pylint: disable = unused-argument
```

Comment:
Disables placed like that will likely cover the whole file. Try to use them on the same line you need disable on